### PR TITLE
chore: add missing package deps & upgrade to webpack-bundle-analyzer@3

### PR DESCRIPTION
**Pull request by TheAlexLichter**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by clarkdo — CHANGES_REQUESTED** [review]
No `yarn.lock` change.

File: packages/nuxt-start/package.json [hunk 1]
```diff
@@ -67,12 +67,12 @@
...
     "serialize-javascript": "^1.5.0",
     "serve-static": "^1.13.2",
-    "compression": "^1.7.3",
     "connect": "^3.6.5",
     "launch-editor-middleware": "^2.2.1",
     "server-destroy": "^1.0.1",
     "chalk": "^2.4.1",
     "esm": "^3.0.80",
-    "ip": "^1.1.5"
+    "ip": "^1.1.5",
```
Comment (clarkdo) [inline packages/nuxt-start/package.json]: Is this change necessary ? 😄

Comment (manniL) [reply packages/nuxt-start/package.json]: Oh well 🙈 
Of course not!

**Comment by manniL** [comment]
Good point @clarkdo!
Will add yarn.lock changes when I come home.

File: packages/nuxt-start/package.json (hunk 1 above)
Comment (clarkdo) [reply packages/nuxt-start/package.json]: Maybe I missed sth, if you generated this by `yarn build`, I think it's fine.

**Review by clarkdo — APPROVED** [review]
